OPERATORS.HTML

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <H2> The Operator </H2>
+  <p id="z1"></p>
+  <p id="z2"></p>
+  <p id="z3"></p>
+  <p id="z4"></p>
+  <p id="z5"></p>
+  <p id="z6"></p>
+  <script>
+    let x = 6;
+    let y = 4;
+    let z1 = x + y;
+    let z2 = x - y;
+    let z3 = x * y;
+    let z4 = x % y;
+    let z5 = ++x;
+    let z6 = --x;
```

Comment:
why did we use `let` instead of `const`?

Reply:
I think we can use both